src/service/network.ts

```diff
@@ -89,6 +92,14 @@ export class Wifi extends Service {
             ]);
             this._activeAp();
         }
+        const ip4Config = this._device.ip4_config;
+        const ip6Config = this._device.ip6_config;
+
+        if ((this._device) && (ip4Config))
+            ip4Config.connect('notify::addresses', () => this.changed('ipv4-address'));
+
+        if ((this._device) && (ip6Config))
+            ip6Config.connect('notify::addresses', () => this.changed('ipv6-address'));
```

Comment:
I don't know much about networks, but is the ip6_config object on the device permanent? becaue if its not this would need to be disconnected when it changes